plan/new.go

```diff
@@ -52,7 +52,23 @@ func buildTestCases(plan *Plan) []TestCase {
 				Plan:      plan,
 				Client:    client,
 				Arguments: testArgs,
+				Skip:      false,
 			}
+
+			for _, filter := range behavior.SkipFilters {
+				// Each filter should be a complete match
+				isCompleteMatch := true
+				for key, value := range filter.AxisMatches {
+					if (key == behavior.ClientAxis && value != client) ||
```

Comment:
you are only testing the client axis here, not all axes.

Also filter.AxisMatches is a map, but you don't seem to use any properties of the map either.